src/main/java/org/spongepowered/common/event/tracking/IPhaseState.java

```diff
@@ -906,4 +906,7 @@ default boolean isConvertingMaps() {
     default boolean allowsGettingQueuedRemovedTiles() {
         return false;
     }
+    
```

Comment:
Can you document what this method is intending to do? I know that not all of the methods in `IPhaseState` are well documented, but it's best when adding new ones that they are documented.